packages/jsapi-utils/src/TableUtils.test.ts

```diff
@@ -279,7 +693,9 @@ describe('quick filter tests', () => {
     it('handles empty cases', () => {
       const column = makeFilterColumn();
 
+      // @ts-expect-error text is generally a string
       expect(TableUtils.makeQuickNumberFilter(column, null)).toBe(null);
+      // @ts-expect-error text is generally a string
       expect(TableUtils.makeQuickNumberFilter(column, undefined)).toBe(null);
```

Comment:
Instead of writing these test cases for invalid input (which should be caught by TypeScript anyway), just take out the lines in `makeQuickNumberFilter` where we check `text == null` (since it can never happen).